### PR TITLE
P0 button image input

### PR DESCRIPTION
**Pull request by mayuresh-tech**
Work done:
Image, Buttons, and Form added (Responsive)

To do:
Start working on the card component

Bugs:
Sidebar:
If the user tests responsive then the sidebar elements get invisible.
aside needs to be given static height (for now set to 500vh)

### CODE REVIEW

**Review by kushalsourav — APPROVED** [review]
looks good to merge

File: components/button/button.css [hunk 1]
```diff
@@ -0,0 +1,288 @@
...
+.no-bg.btn-icon-text-outline {
+  cursor: pointer;
+  margin: 1rem 1rem;
+  text-decoration: none;
+  background-color: var(--color-white);
+  padding: 0.5rem 1rem;
+  border: var(--btn-color-primary) 1px solid;
+  color: var(--btn-color-primary);
+  font-size: 1rem;
+  border-radius: 0.3rem;
```
Comment (deekshithmd) [inline components/button/button.css:72]: try to use px for border radius

File: components/button/button.css [hunk 2]
```diff
@@ -0,0 +1,288 @@
...
+}
+
+.btn-outline-secondary {
+  text-decoration: none;
+  background-color: var(--color-white);
+  padding: 0.5rem 1rem;
+  border: var(--btn-color-secondary-text) 1px solid;
+  color: var(--btn-color-secondary-text);
+  font-size: 1rem;
+  border-radius: 0.3rem;
```
Comment (deekshithmd) [inline components/button/button.css:125]: same here, using px will be good

**Review by deekshithmd — APPROVED** [review]

File: components/input/input.css [hunk 3]
```diff
@@ -0,0 +1,113 @@
...
+  margin: 0rem 0rem 0rem 1rem;
+  margin-bottom: 0.3rem;
+  padding: 0.5rem;
+}
+
+.validate input:focus {
+  margin: 0rem 0rem 0rem 1rem;
+  margin-bottom: 0.3rem;
+  padding: 0.5rem;
+}
```
Comment (m26unkwn) [inline components/input/input.css:110]: you can use margin-left property here

**Review by m26unkwn — APPROVED** [review]
Otherwise everything is good to merge.

**Pull request merged**